proxy/cache.go

```diff
@@ -22,7 +22,7 @@ type cache struct {
 	sync.RWMutex               // lock
 }
 
-func (c *cache) Get(request *dns.Msg) (*dns.Msg, bool) {
+func (c *cache) Get(request *dns.Msg) (msg *dns.Msg, isNotExpired bool) {
```

Comment:
It's generally better to not add negatives to names of boolean variables, so `isExipred` sounds better.

Reply:
fixed